ethereum-rpc/src/constructors/express/routes/api/v1/uniswap.js

```diff
@@ -105,67 +98,40 @@ module.exports = ({
           return res.status(400).json({ errors: errors.array() });
         }
         const { address = "*" } = req.params;
-        const { privateKey = "", amount = "", tokenFrom = "", tokenTo = "" } = { ...req.query };
-        const tokenType = tokenFrom.toString().toUpperCase();
+        const { privateKey = "", amount = "", network = "", tokenFrom = "", tokenTo = "", tokenFromAddress = "" } = { ...req.query };
+        if (tokenFrom === tokenTo) {
+          res.status(500).json({ error: "tokenFrom and tokenTo cannot be same" })
+        }
+        const networkType = network.toString().toUpperCase();
         const routerAbi = abi.UNISWAP_ROUTER.abi
         const tokenAbi = abi.APIS_ERC20
-        const chainIds = ChainId.RINKEBY;
-
-        const tokenAddressRinkebyDai = '0xc7ad46e0b8a400bb3c915120d284aafba8fc4735'
-        const tokenAddressRinkebyUniswap = '0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984';
-        const tokenAddressRinkebyMaker = '0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85';
-        const tokenAddressRinkebyRouter = '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'
-
-        const DAI = await Fetcher.fetchTokenData(chainIds, tokenAddressRinkebyDai);
-        const UNISWAP = await Fetcher.fetchTokenData(chainIds, tokenAddressRinkebyUniswap);
-        const MAKER = await Fetcher.fetchTokenData(chainIds, tokenAddressRinkebyMaker);
+        const chainIds = chainNetwork()[networkType]
+        const tokenAddressRouter = UNISWAP_ROUTER_ADDRESS
```

Comment:
We can just use the constant, I do not see a need to rename `UNISWAP_ROUTER_ADDRESS`